Makefile

```
@@ -83,9 +79,25 @@ test-docs: ## Run documentation tests
 
 # --- Integration testing -------------------------------------------------------------------------
 
+.PHONY: start-node
+start-node: ## Start the testing node server
+	RUST_LOG=none cargo run --release --package node-builder & echo $$! > .node.pid; \
+	sleep 4; \
+	if ! ps -p $$(cat .node.pid) > /dev/null; then \
+		echo "Failed to start node server"; \
+		rm -f .node.pid; \
+		exit 1; \
+	fi; \
+	rm -f .node.pid
```

Awesome feature! This should make the CI fail early if the node isn't working, which was a problem we had in the past. Maybe we should move this script to the `scripts` directory. This way we can also remove all those `\`.

---

Thanks!

Yes, you are right. I'll move it there.